doc/contribute/CONTRIBUTE.md

```diff
@@ -24,6 +24,38 @@ Once the changes in your pull request are approved by one of our reviewers, they
 
 Find here [a guide](../setup/SETUP.md) on how to setup your local development environment.
 
+## CSS Guidelines
+
+We are using [Scss](https://sass-lang.com) to write modular, reusable css.
+
+We have a couple of global scss files in `webapp/content` but encourage [component dependent css with angular's styleUrls](https://angular.io/guide/component-styles).
+
+The [bootstrap css](https://getbootstrap.com/) is imported into Artemis, however we discourage using it in html.
+We encourage using the Scss [extend](https://sass-lang.com/documentation/at-rules/extend) mechanism if bootstrap styles are used.
+
+Discouraged html styling:
+`<div class="d-flex">some content</div>`
```

Comment:
Why not using bootstrap helper classes? especially with margins and paddings it will ensure that the same distance is used in all elements

Reply:
We discussed this in the last developer meeting: The mixture of global styles, component styles and also bootstrap classes in html becomes hard to maintain.

Also the bootstrap classes are piling up, e.g. `d-flex flex-flow-column mr-2 p-2` becomes hard to read.

Reply:
Also please see the section below your comment, we still encourage using bootstrap styles in the extend mechanism of scss.